app/dashboard/models.py

```diff
@@ -2207,6 +2217,16 @@ def related_to(self, profile):
         return posts
 
 
+class ActivityManager(models.Manager):
+    """Enables changing the default queryset function for Activities."""
+
+    def get_queryset(self):
+        if settings.ENV == 'prod':
+            return super().get_queryset().filter(Q(bounty=None) | Q(bounty='') | Q(bounty__network='mainnet'))
```

Comment:
this was failing everyplace where activity is fetched on production I just removed the 'prod' condition locally and seems is replicable on localhost.